airflow/providers/microsoft/azure/sensors/wasb.py

```diff
@@ -62,6 +65,64 @@ def poke(self, context: Context):
         return hook.check_for_blob(self.container_name, self.blob_name, **self.check_options)
 
 
+class WasbBlobAsyncSensor(WasbBlobSensor):
+    """
+    Polls asynchronously for the existence of a blob in a WASB container.
+
+    :param container_name: name of the container in which the blob should be searched for
+    :param blob_name: name of the blob to check existence for
+    :param wasb_conn_id: the connection identifier for connecting to Azure WASB
+    :param poll_interval:  polling period in seconds to check for the status
```

Comment:
```suggestion
    :param poke_interval:  polling period in seconds to check for the status
```

Reply:
WDYT about also adding `timeout` to the docstring too? Just thinking users wouldn't have to traverse parent classes to figure out this param can be set.

Reply:
Done